Templates/gRPC-Template/.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Auto detect text files and perform LF normalization
```

Comment:
How will this affect windows developers? ahem, @johanfol

Reply:
Also, we're forcing the same thing in the .editorconfig. It should be fine in windows but confirmation is needed.

Reply:
This reformats the files, so when opening in vsCode, it just reformats it. So on commit it at least should have LF's for you okes.